tag_hierarchy/commands/populategraph.cpp

```diff
@@ -0,0 +1,54 @@
+//
+// Created by Petter Moe Kvalvaag on 2019-10-03.
+//
+
+#include "tag_hierarchy/tag_hierarchy.h"
+#include "tag_hierarchy/commands/populategraph.h"
+
+PopulateGraph populategraph_(std::string("populate_graph"));
```

Comment:
Very happy to see this after discussion we had yesterday :) No macro!

Reply:
😁